File: ios/SwellSeeker/SwellSeeker/Models/Beach.swift
import Foundation

struct Beach: Identifiable, Codable {
    let id: Int
    let beachName: String
    let town: String
    let state: String
    let lat: Double
    let long: Double
    let beachAngle: Double
    let stationId: String?
    
    enum CodingKeys: String, CodingKey {
        case id
        case beachName = "beach_name"
        case town
        case state
        case lat
        case long
        case beachAngle = "beach_angle"
        case stationId = "station_id"
    }
}

struct BeachList: Codable {
    let beaches: [Beach]
} 

File: backend/app/schemas/beach.py
from pydantic import BaseModel
from typing import Optional
from datetime import datetime

class BeachBase(BaseModel):
    beach_name: str
    town: str
    state: str
    lat: float
    long: float
    beach_angle: float
    station_id: str

class BeachCreate(BeachBase):
    pass

class Beach(BeachBase):
    id: int
    created_at: datetime
    updated_at: Optional[datetime] = None
    
    class Config:
        from_attributes = True

class BeachList(BaseModel):
    beaches: list[Beach] 